.github/workflows/main.yml

```diff
@@ -1,6 +1,8 @@
 name: Windows RTools 3.5
 
 on:
+  pull_request:
+    branches: [ develop, master ]
```

Comment:
This one is obvious. In addition to running on merges this will now run these tests also on PRs to develop or master (the latter are rare - patch fixes for release, but can happen).